<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>databaseview</class>
 <widget class="QMainWindow" name="databaseview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1220</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="displayCard_front">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>221</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="displayCard_back">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>410</y>
      <width>181</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="ptentry_back">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>240</y>
      <width>311</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="ptentry_front">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="txtlabel_back">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtLabel_front">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Front</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtLabel_insertFlashcards">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Create flashcard entry</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInsertCard">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert to sql database</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>380</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Showing card at 1 (Front &amp; Back)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowCards">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show cards in debug log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBackMain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Back to main</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1220</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
